marc_to_solr/lib/princeton_marc.rb

```diff
@@ -638,13 +638,16 @@ def process_holdings record # rubocop:disable Metrics/AbcSize, Metrics/Cyclomati
     is_alma = alma_code?(field['8'])
     is_scsb = scsb_doc?(record['001'].value)
     field.subfields.each do |s_field|
-      if s_field.code == '8'
-        holding_id = s_field.value if is_alma || is_scsb
+      # Index holdings from Princeton records
+      if s_field.code == '8' && is_alma
+        holding_id = s_field.value
+      # Index holdings from SCSB
       elsif s_field.code == '0'
         holding_id = s_field.value
```

Comment:
Should this have the `is_scsb` check?

Reply:
No it shouldn't. The scsb check was added to make sure that we index the scsb because we have the check for the alma valid princeton record.